src/api/chassis/model/hDriveModel.cpp

```diff
@@ -144,6 +144,42 @@ void HDriveModel::arcade(const double iforwardSpeed, const double iyaw, const do
   middleMotor->moveVelocity(0);
 }
 
+void HDriveModel::curvature(const double iforwardSpeed, 
+                            const double icurvature, 
+                            const double ithreshold) {
+  // This code is adapted from WPIlib. All credit goes to them. Link:
+  // https://github.com/wpilibsuite/allwpilib/blob/main/wpilibc/src/main/native/cpp/drive/DifferentialDrive.cpp#L49
```

Comment:
This should probably get updated to a permalink like https://github.com/wpilibsuite/allwpilib/blob/af7985e46cc3fd6eb01c1243c7f1f95b92ea5ddc/wpilibc/src/main/native/cpp/drive/DifferentialDrive.cpp#L49 in case the WPILib file changes and the current link points to a different part of the code.

Reply:
The link have been replaced. I also went ahead and replaced the links in functions like tank() and driveVector() because they are all dead.